tools/cockpit.spec

```diff
@@ -71,7 +71,10 @@ BuildRequires: pcp-libs-devel
 BuildRequires: gdb
 
 %if %{defined gitcommit}
+# on fedora 24, nodejs provides npm
+%if 0%{?fedora} > 0 && 0%{?fedora} < 24
```

Comment:
Should be fine. I have a branch that removes almost all the gitcommit stuff from cockpit.spec anyway